.travis.yml

```diff
@@ -2,6 +2,8 @@ script: "bundle exec rake neo4j:install[community-2.3.0] neo4j:start default --t
 language: ruby
 cache: bundler
 sudo: false
+jdk:
+  - oraclejdk8
```

Comment:
I'm not sure about this case, but I know if you wanted to add Neo4j 3.0 you need to use JDK 8.

It took me a couple of weeks on-and-off getting Neo4j 3.0 working on Travis, so if you'd like to do it I would consult the [`neo4j` gem's `.travis.yml`](https://github.com/neo4jrb/neo4j/blob/master/.travis.yml), particularly the `echo` lines at the top which configure the memory and the `sleep` which allows Neo4j to start up before the tests run